scripts/checkbot.pl

```diff
@@ -95,11 +126,27 @@ ($)
 
 }
 
-# TODO: randomize the alerts, not to have always the same products
-my $cursor = get_products_collection()->query({})->fields({ code => 1 });;
-my $count = $cursor->count();
+my $query = {};
+
+# If --country is specified, build the query with the country
+# TODO: hold the case where 'countries' => 'France,en:Belgium'
```

Comment:
What do you think about querying 'countries_tags'? That field contains an array, so that searching for `{"countries_tags":"en:canada"}` also returns a product with `"countries_tags" : [ "en:canada", "en:france" ]`.

Reply:
Thanks. Made a correction and a little enhancement.